packages/apollo/src/commands/service/check.ts

```diff
@@ -147,6 +154,32 @@ ${
 `;
 }
 
+export function formatCompositionErrorsMarkdown({
+  compositionErrors,
+  graphName,
+  serviceName,
+  tag
+}: {
+  compositionErrors: CompositionErrors;
+  graphName: string;
+  serviceName: string;
+  tag: string;
+}): string {
+  return `
+### Apollo Service Check
+🔄 Validated graph composition on schema tag \`${tag}\` for service \`${serviceName}\` on graph \`${graphName}\`.
+❌ Found **${compositionErrors.length} composition errors**
+
+| Service   | Field     | Message   |
+| --------- | --------- | --------- |
+${compositionErrors
+  .map(
+    ({ service, field, message }) => `| ${service} | ${field} | ${message} |`
```

Comment:
Is there a way to make these line up with the header? I think in the human readable sections we use the heroku table.